sam local start-api returns panic: start-api flag redefined: p

have written a simple hello world api.


```
'use strict';

exports.handler = (event, context, callback) => {
    console.log("event querystrings: ", event.queryStringParameters);
    callback(null, {
        statusCode: 200,
        headers: {},
        body: "event querystrings: " + JSON.stringify(event.queryStringParameters)
    });
}
```


with template.yaml file

```
AWSTemplateFormatVersion: '2010-09-09'
Transform: AWS::Serverless-2016-10-31
Description: Simple Hello world type example
Resources:
    hello:
        Type: AWS::Serverless::Function
        Properties:
            Handler: hello.handler
            Runtime: nodejs6.10
            CodeUri: s3://sam-workshop
            Events:
                Api:
                    Properties:
                    Path: /hello
                    Method: get
```


using a event that was generated by sam.

I get this error


```
sam local start-api
start-api flag redefined: p
panic: start-api flag redefined: p

goroutine 1 [running]:
flag.(*FlagSet).Var(0xc42021cfc0, 0x1c7a5a0, 0xc42032263c, 0x187254a, 0x1, 0x188e7b1, 0x61)
        /usr/local/go/src/flag/flag.go:793 +0x420
flag.(*FlagSet).BoolVar(0xc42021cfc0, 0xc42032263c, 0x187254a, 0x1, 0x10e0200, 0x188e7b1, 0x61)
        /usr/local/go/src/flag/flag.go:566 +0x70
flag.(*FlagSet).Bool(0xc42021cfc0, 0x187254a, 0x1, 0x0, 0x188e7b1, 0x61, 0x1)
        /usr/local/go/src/flag/flag.go:579 +0x7f
github.com/awslabs/aws-sam-local/vendor/github.com/codegangsta/cli.BoolFlag.ApplyWithError.func1(0x187254a, 0x1)
        /Users/podgeobrien/src/github.com/awslabs/aws-sam-local/vendor/github.com/codegangsta/cli/flag.go:352 +0xa7
github.com/awslabs/aws-sam-local/vendor/github.com/codegangsta/cli.eachName(0x1872539, 0x12, 0xc4201e1ea0)
        /Users/podgeobrien/src/github.com/awslabs/aws-sam-local/vendor/github.com/codegangsta/cli/flag.go:94 +0xb5
github.com/awslabs/aws-sam-local/vendor/github.com/codegangsta/cli.BoolFlag.ApplyWithError(0x1872539, 0x12, 0x188e7b1, 0x61, 0x18729f6, 0x13, 0x0, 0x0, 0xc42021cfc0, 0x1, ...)
        /Users/podgeobrien/src/github.com/awslabs/aws-sam-local/vendor/github.com/codegangsta/cli/flag.go:353 +0xef
github.com/awslabs/aws-sam-local/vendor/github.com/codegangsta/cli.(*BoolFlag).ApplyWithError(0xc42032c000, 0xc42021cfc0, 0xc42032c000, 0x2b0c0d0)
        <autogenerated>:21 +0x66
github.com/awslabs/aws-sam-local/vendor/github.com/codegangsta/cli.flagSet(0x186c2f4, 0x9, 0xc420334240, 0xa, 0x12, 0xc420334240, 0x9, 0x12)
        /Users/podgeobrien/src/github.com/awslabs/aws-sam-local/vendor/github.com/codegangsta/cli/flag.go:80 +0xfa
github.com/awslabs/aws-sam-local/vendor/github.com/codegangsta/cli.Command.Run(0x186c2f4, 0x9, 0x0, 0x0, 0x0, 0x0, 0x0, 0x18908e0, 0x3c9, 0x0, ...)
        /Users/podgeobrien/src/github.com/awslabs/aws-sam-local/vendor/github.com/codegangsta/cli/command.go:104 +0xbc
github.com/awslabs/aws-sam-local/vendor/github.com/codegangsta/cli.(*App).RunAsSubcommand(0xc42011c9c0, 0xc42023de00, 0x0, 0x0)
        /Users/podgeobrien/src/github.com/awslabs/aws-sam-local/vendor/github.com/codegangsta/cli/app.go:374 +0xacc
github.com/awslabs/aws-sam-local/vendor/github.com/codegangsta/cli.Command.startApp(0x1869a05, 0x5, 0x0, 0x0, 0x0, 0x0, 0x0, 0x188d133, 0x47, 0x0, ...)
        /Users/podgeobrien/src/github.com/awslabs/aws-sam-local/vendor/github.com/codegangsta/cli/command.go:293 +0x8a3
github.com/awslabs/aws-sam-local/vendor/github.com/codegangsta/cli.Command.Run(0x1869a05, 0x5, 0x0, 0x0, 0x0, 0x0, 0x0, 0x188d133, 0x47, 0x0, ...)
        /Users/podgeobrien/src/github.com/awslabs/aws-sam-local/vendor/github.com/codegangsta/cli/command.go:93 +0x14c7
github.com/awslabs/aws-sam-local/vendor/github.com/codegangsta/cli.(*App).Run(0xc42011c820, 0xc4200101b0, 0x3, 0x3, 0x0, 0x0)
        /Users/podgeobrien/src/github.com/awslabs/aws-sam-local/vendor/github.com/codegangsta/cli/app.go:250 +0x7d0
main.main()
        /Users/podgeobrien/src/github.com/awslabs/aws-sam-local/main.go:338 +0x1f97
```
any one have any idea whats going on here?

--- a/main.go
+++ b/main.go
@@ -96,13 +96,12 @@ func main() {
 								"you must mount the path where the SAM file exists on the docker machine and modify this value to match the remote machine.",
 							EnvVar: "SAM_DOCKER_VOLUME_BASEDIR",
 						},
-						cli.BoolFlag {
-							Name: "skip-pull-image, p",
-							Usage: "Optional. Specify whether SAM should skip pulling down the latest Docker image. Default is false.",
+						cli.BoolFlag{
+							Name:   "skip-pull-image",
+							Usage:  "Optional. Specify whether SAM should skip pulling down the latest Docker image. Default is false.",
 							EnvVar: "SAM_SKIP_PULL_IMAGE",
 						},
-	
-				},
+					},
 				},
 				cli.Command{
 					Name:   "invoke",
@@ -141,12 +140,11 @@ func main() {
 								"you must mount the path where the SAM file exists on the docker machine and modify this value to match the remote machine.",
 							EnvVar: "SAM_DOCKER_VOLUME_BASEDIR",
 						},
-						cli.BoolFlag {
-							Name: "skip-pull-image, p",
-							Usage: "Optional. Specify whether SAM should skip pulling down the latest Docker image. Default is false.",
+						cli.BoolFlag{
+							Name:   "skip-pull-image",
+							Usage:  "Optional. Specify whether SAM should skip pulling down the latest Docker image. Default is false.",
 							EnvVar: "SAM_SKIP_PULL_IMAGE",
 						},
-	
 					},
 				},
 				cli.Command{

